integrated_channels/integrated_channel/tasks.py

```diff
@@ -164,7 +164,7 @@ def transmit_single_subsection_learner_data(username, course_run_id, subsection_
     # Starting Export. N customer is usually 1 but multiple are supported in codebase.
     for enterprise_customer_uuid in enterprise_customer_uuids:
         for channel in channel_utils.get_integrated_channels(
-                {'channel': None, 'enterprise_customer': enterprise_customer_uuid}
+                {'channel': None, 'enterprise_customer': enterprise_customer_uuid, 'assessment_level': True}
```

Comment:
This is really pedantic and in no way a blocker, but can we use either `assessment_level` or `assessment_level_support` consistently? I think it's pretty easy still to just search the actual method to see what the parameter name is and given `assessment_level` is contained within `assessment_level_support` it's still even searchable, but it might make the trail just a little bit easier to follow if the parameter name is the same. It's also an internal parameter afaict (ie we aren't sending "assessment_level" to Canvas) so I don't think there's any reason really for them to not match (unless I'm missing something, in which case ignore me!)

Reply:
hmm yeah, I'm all for consistency. I think this became desynced when I had a pylint error on variable naming conventions... Will see what I can do to make it consistent.